homeassistant/components/zwave_js/__init__.py

```diff
@@ -924,7 +924,8 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     unload_ok = all(await asyncio.gather(*tasks)) if tasks else True
 
-    await async_disable_server_logging_if_needed(hass, entry, driver_events.driver)
+    if hasattr(driver_events, "driver"):
```

Comment:
This is a weird check. I think we should change the last part of the config entry setup like we have done in the matter integration:

https://github.com/home-assistant/core/blob/f899e5159b14a650871bead9b97f79d0c86df5e5/homeassistant/components/matter/__init__.py#L107-L114

Then we can guarantee that the driver attribute is set when the config entry is setup. We can change that in a follow up.

Reply:
this was actually an extra commit that I was using to figure out how to get another unrelated test to not fail. I will refactor this in another PR to be cleaner and will try your suggestion